tests/generate_packageTest.py

```diff
@@ -83,12 +85,67 @@ def test_command(self, call_mock):
         call_mock.assert_called_with(['twine', 'upload', './dist/*'])
 
 
+class get_bear_globTest(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        bear_path = os.path.join('tests', 'conda_package_testfiles',
+                                 'test_conda_bear')
+        shutil.copytree(os.path.join(bear_path, 'git'),
+                        os.path.join(bear_path, '.git'))
+
+    def test_single_bear(self):
+        bear_path = os.path.join('tests', 'conda_package_testfiles',
+                                 'test_conda_bear')
+        result = get_bear_glob(bear_path)
+        self.assertEqual(result, [os.path.join(bear_path, 'TestBear.py')])
+
```

Comment:
I guess I could add a test but this is covered on adrian's part. I added the single bear support